src/file_organizer/undo/rollback.py

```
@@ -276,6 +282,30 @@ def rollback_copy(self, operation: Operation) -> bool:
+
+        try:
+            if not link_path.is_symlink():
+                logger.error(
+                    f"Refusing to rollback symlink operation; destination is not a symlink: {link_path}"
+                )
+                return False
+            link_path.unlink()
+            fsync_directory(link_path)
+            logger.info(f"Successfully rolled back symlink operation {operation.id}")
+            return True
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)